tip-distributor/src/lib.rs

```diff
@@ -460,104 +470,212 @@ pub fn derive_tip_distribution_account_address(
     )
 }
 
+pub const MAX_RETRIES: usize = 5;
+pub const FAIL_DELAY: Duration = Duration::from_millis(100);
+
+/// Returns unprocessed transactions, along with fail count
+pub async fn sign_and_send_transactions_with_retries_multi_rpc(
+    signer: &Arc<Keypair>,
+    blockhash_rpc_client: &Arc<RpcClient>,
+    rpc_clients: &Arc<Vec<Arc<RpcClient>>>,
```

Comment:
should make sure we used confirmed or finalized here. finalized will make it much slow though

if we used processed, might end up false confirming a block that gets forked out

Reply:
they all use confirmed here:
https://github.com/jito-labs/jito-solana/blob/fda99db3060d4e2ac9dbb558f8944ac5ffa0c01a/tip-distributor/src/claim_mev_workflow.rs#L93-L102